x-pack/plugins/security_solution/common/endpoint/schema/actions.ts

```diff
@@ -15,7 +15,7 @@ import {
 
 const BaseActionRequestSchema = {
   /** A list of endpoint IDs whose hosts will be isolated (Fleet Agent IDs will be retrieved for these) */
-  endpoint_ids: schema.arrayOf(schema.string(), { minSize: 1 }),
+  endpoint_ids: schema.arrayOf(schema.string({ minLength: 1 }), { minSize: 1 }),
```

Comment:
Where are you validating for empty strings? I don't see that code and would have expected it here as part of the `schema.arrayOf()`.

Reply:
it's part of the test. But with this change it would mean that you can't pass in `[]`   or `[""]`

Reply:
Ohhh. I see. I'm sorry. I though the `minLength` was added to the array definition. I see now that its against the `schema.string()`.

Note that you can (maybe) still create a request with "empty" ids - example: `[ "   "]`. You might still need to add a `validate()` function here to the `.arrayOf()`.

Also, I think that same setup needs to be added to the other properties here - like 'alert_ids` and `case_ids`

Reply:
for endpoint_ids and `execute` `parameters.command` 8f1e33d4873d4a7658336701ed0decc46da5a564

Reply:
for case and aler ids. af23c13863bbd1c499bd2722eed9dbd688b761d2